### DISCUSSION TITLE
Fully use IL interpreter?

### DISCUSSION THREAD

**Discussion by heiher** [dotnet/runtime#44064] [Q&A] [answered]
Even under the configuration of `COMPlus_ForceInterpreter=1`, there is still JIT compilation. Is there a way to disable all JIT and use the interpreter to execute them all?

@jkotas @yuchong-pan 

Thank you.

**Comment by jkotas** [comment, marked as answer]
CoreCLR interpreter is incomplete experiment and it is not compiled into the shipping CoreCLR binaries.

You may want to look at https://github.com/dotnet/runtime/tree/master/src/mono. Mono has a complete interpreter.

**Reply by heiher** [reply]
Thank you.
